pom.xml

```diff
@@ -94,6 +98,22 @@
 						</execution>
 					</executions>
 				</plugin>
+				<plugin>
+					<groupId>org.codehaus.mojo</groupId>
+					<artifactId>cobertura-maven-plugin</artifactId>
+					<version>2.7</version>
+					<configuration>
+						<format>xml</format>
+						<maxmem>256m</maxmem>
+						<!-- aggregated reports for multi-module projects -->
+						<aggregate>true</aggregate>
+						<instrumentation>
+							<excludes>
+								<exclude>com/ibm/g11n/pipeline/maven/HelpMojo.class</exclude>
```

Comment:
this is generated code and shouldn't be covered. (in any event, its source is harder for cobertura to find)